sdks/python/apache_beam/runners/interactive/pipeline_instrument.py

```diff
@@ -337,6 +358,7 @@ def cacheables(pcolls_to_pcoll_id):
   """
   pcoll_version_map = {}
   cacheables = {}
+  cacheable_var_by_pcoll_id = {}
```

Comment:
Can't comment on already-committed lines, so commenting here.
I didn't get what a "cache desired PCollection" is. cache-desired PCollection looks slightly better, but still doesn't tell what it is. PCollections that need to be cached?

Reply:
Yes, it is actually PCollections that need to be cached. I thought it was a little bit wordy and chose the shorter form. I'll rephrase it to "PCollections that need to be cached" instead for clarity.